packages/core/src/ceramic.ts

```diff
@@ -67,7 +67,7 @@ import { LevelKVFactory } from './store/level-kv-factory.js'
 const METRICS_CALLER_NAME = 'js-ceramic'
 const DEFAULT_CACHE_LIMIT = 500 // number of streams stored in the cache
 const DEFAULT_QPS_LIMIT = 10 // Max number of pubsub query messages that can be published per second without rate limiting
-const DEFAULT_MULTIQUERY_TIMEOUT_MS = 30 * 1000
+const DEFAULT_MULTIQUERY_TIMEOUT_MS = 3 * 1000
```

Comment:
I'm not positive, but I believe when this timeout fires it will only fail the multiquery request but not clean up the work happening in the background in the execution queue.  I think probably you want to pair this with also turning down the IPFS timeouts in the dispatcher.

Reply:
ok i had made a separate PR for those since i only saw the multiquery errors in the logs.  we could do https://github.com/ceramicnetwork/js-ceramic/pull/3285 instead.  I thought that might require more testing if its hitting more locations.